release-notes/6.0/preview/6.0.0-preview.4-runtime-deps.json

```diff
@@ -0,0 +1,1035 @@
+        {
+          "name": "System.Drawing.Common",
+          "type": "NuGetPackage",
+          "platformDependencies": [
+            {
+              "name": "libcups2",
+              "dependencyType": "LinuxPackage",
+              "usage": "printing"
+            },
+            {
+              "name": "libgdiplus0:6.0.1",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libx11-dev",
+              "dependencyType": "LinuxPackage",
+              "usage": "xwindows"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "rid": "ubuntu",
+      "components": [
+        {
+          "name": "Microsoft.NETCore.App",
+          "type": "Framework",
+          "platformDependencies": [
+            {
+              "name": "libc6",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libgcc1",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libgssapi-krb5-2",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libicu60",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "liblttng-ust0",
+              "dependencyType": "LinuxPackage",
+              "usage": "diagnostics"
+            },
+            {
+              "name": "libssl1.1",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libstdc++6",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "tzdata",
+              "dependencyType": "LinuxPackage",
+              "usage": "localization"
+            },
+            {
+              "name": "zlib1g",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            }
+          ]
+        },
+        {
+          "name": "System.DirectoryServices.Protocols",
+          "type": "NuGetPackage",
+          "platformDependencies": [
+            {
+              "name": "libldap-2.4-2",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            }
+          ]
+        },
+        {
+          "name": "System.Drawing.Common",
+          "type": "NuGetPackage",
+          "platformDependencies": [
+            {
+              "name": "libcups2",
+              "dependencyType": "LinuxPackage",
+              "usage": "printing"
+            },
+            {
+              "name": "libgdiplus:6.0.1",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libx11-dev",
```

Comment:
Do we really have dependency on the `-dev` package? The `-dev` package typically contain static libraries, required for C/C++ compilation and linking. They should not be needed to by .NET runtime.

Reply:
That's what I thought but it is required right now.  After looking at what gets installed I see this:

Installing libx11-6:
* /usr/lib/x86_64-linux-gnu/libX11.so.6
* /usr/lib/x86_64-linux-gnu/libX11.so.6.3.0

If attempting to execute a code path that requires the X11 library with just `libx11-6` installed, you get this error:

```
System.DllNotFoundException: 'Unable to load shared library 'libX11' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: liblibX11: cannot open shared object file: No such file or directory'
```

It works after installing `libx11-dev`.  The additional file that `libx11-dev` provides is the following:
* /usr/lib/x86_64-linux-gnu/libX11.so

Should I log an issue to be able to search for the additional versioned .so file in order to remove the dependency on `-dev`?

Reply:
> Should I log an issue to be able to search for the additional versioned .so file in order to remove the dependency on -dev?

IMO, yes. The `-dev`/`-devel` packages are not generally installed on end-user systems. They are only used by developers.

Reply:
I've logged https://github.com/dotnet/runtime/issues/51404 to remove the dependency on the developer package.